docs/standards.md

```diff
@@ -0,0 +1,167 @@
+## SG2 Standards Checklist
+
+Since SG2 is a living design library that is home to reusable patterns, it's important for all contributing developers to attempt to create patterns the same way. 
+
+Any new work contributed into SG2 must meet the following standards:
+
+### PatternLab Organization
+
+#### How patterns are organized:
+- Patterns should be listed alphabetically in the pattern list. No numbers (e.g. `atoms/forms/text-input` instead of `00-atoms/10-forms/05-text`). 
+- Talk about where SCSS and JS files should go (in a separate .scss directory?), as well as Twig and .md files (no subdirectories?). We may still need to make a decision about this.
```

Comment:
This should be a discussion. My proposal is that scss and js files be added in an `assets` directory that mimics the `_patterns` directory. Meaning that if I have `breadcrumbs.twig`, `breadcrumbs.json` and `breadcrumbs.md` files at `styleguide/source/_patterns/organisms/` then I would expect a corresponding scss file to live at `styleguide/source/assets/scss/organisms/_breadcrumbs.scss` and the corresponding js (if necessary) to live at `styleguide/source/assets/js/organisms/breadcrumbs.js`.

Reply:
Per FED today the directory structure inside each pattern type (atoms, etc.) should be flat; patterns shouldn't go inside grouped subfolders. So like this:

Atoms > Heading
Atoms > Link
Atoms > Whatever 

rather than

Atoms > Text > Heading, etc.

Main reasoning for this is that adding the subdirectories makes referencing the patterns by path (which is necessary for Drupal templating) a pain.